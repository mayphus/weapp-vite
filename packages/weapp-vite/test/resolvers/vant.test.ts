import vantJson from '@/auto-import-components/resolvers/json/vant.json'

describe('vant', () => {
  it('vant default', () => {
    expect(vantJson).toMatchSnapshot()
  })
})
