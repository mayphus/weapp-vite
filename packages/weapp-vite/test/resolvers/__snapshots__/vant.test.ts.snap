// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`vant > vant default 1`] = `
[
  "action-sheet",
  "area",
  "button",
  "calendar",
  "card",
  "cascader",
  "cell",
  "cell-group",
  "checkbox",
  "checkbox-group",
  "circle",
  "col",
  "collapse",
  "collapse-item",
  "config-provider",
  "count-down",
  "datetime-picker",
  "definitions",
  "dialog",
  "divider",
  "dropdown-item",
  "dropdown-menu",
  "empty",
  "field",
  "goods-action",
  "goods-action-button",
  "goods-action-icon",
  "grid",
  "grid-item",
  "icon",
  "image",
  "index-anchor",
  "index-bar",
  "info",
  "loading",
  "nav-bar",
  "notice-bar",
  "notify",
  "overlay",
  "panel",
  "picker",
  "picker-column",
  "popup",
  "progress",
  "radio",
  "radio-group",
  "rate",
  "row",
  "search",
  "share-sheet",
  "sidebar",
  "sidebar-item",
  "skeleton",
  "slider",
  "stepper",
  "steps",
  "sticky",
  "submit-bar",
  "swipe-cell",
  "switch",
  "tab",
  "tabbar",
  "tabbar-item",
  "tabs",
  "tag",
  "toast",
  "transition",
  "tree-select",
  "uploader",
]
`;
